tests/IntegrationTestDefs.js

```diff
@@ -80,7 +99,17 @@ gpii.tests.app.testDefs = {
     sequence: [{ // Test the menu that will be rendered
         event: "{that gpii.app.menu}.events.onCreate",
         listener: "gpii.tests.app.testInitialMenu"
-    }, { // Test menu after key in
+    }, [ // PCP window tests
```

Comment:
This is a good start at extending the existing tests - but we will need substantial new test infrastructure to verify at least the basics of the rendered UI - e.g. that it contains some kind of populated HTML element for each setting, that they can be operated to initiate a settings change message, etc.